arviz/tests/test_plots.py

```diff
@@ -233,15 +233,18 @@ def test_plot_parallel_raises_valueerror(df_trace):  # pylint: disable=invalid-n
 
 
 @pytest.mark.parametrize("model_fit", ["pymc3_fit", "stan_fit"])
-def test_plot_parallel(models, model_fit):
+@pytest.mark.parametrize("norm_method", [None, "normal", "minmax", "rank"])
+def test_plot_parallel(models, model_fit, norm_method):
     obj = getattr(models, model_fit)
-    assert plot_parallel(obj, var_names=["mu", "tau"])
+    assert plot_parallel(obj, var_names=["mu", "tau"], norm_method=norm_method)
 
 
-def test_plot_parallel_exception(models):
+@pytest.mark.parametrize("var_names", [None, "mu", ["mu", "tau"]])
+@pytest.mark.parametrize("norm_method", ["foo"])

```

Comment:
This one is confusing to me. If there is only one thing you're testing for why not put this directly line 247?

Sort of like this `assert plot_parallel(models.pymc3_fit, var_names=var_names, norm_method='foo)`

Reply:
Don't know why this did not cross my mind. Anyway, I have changed it now.